pkgs/net-doc/net/scribblings/osx-ssl.scrbl

```diff
@@ -3,11 +3,11 @@
           (for-label net/osx-ssl
                      openssl))
 
-@title[#:tag "osx-ssl"]{OS X Native SSL: Secure Communication}
+@title[#:tag "osx-ssl"]{macOS Native SSL: Secure Communication}
```

Comment:
With this, someone searching for "OS X SSL" will not get this result, whereas they would now. Could you add the old title to the index separately?